literature.bib

```diff
@@ -22922,6 +23048,20 @@ @inproceedings{M02
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2001 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{MFH:OOPSLA01,
+	author = {McDirmid, Sean and Flatt, Matthew and Hsieh, Wilson C.},
+	title = {{Jiazzi: New-Age Components for Old-Fasioned Java}},
```

Comment:
Fashioned?

Reply:
You wouldn't believe but this actually a typo in the ACL DL and hence in Google Scholar. It is written "Fashioned" on the actual paper though (phew). Now its the question:
Do we correct the typo here? Google Scholar and co might not be able to resolve a citation correctly then?

Reply:
@tthuem Should we fix typos in paper titles even when they are "official"? Personally, I would fix it. Looks wrong on our papers otherwise and matches the title on the actual paper. (ACM DL frequently gets case and spaces wrong, and we don't preserve those either)

Reply:
Fine with me to fix those

Reply:
fixed